transcom/mymove
# B-18182 remove old personally procured move handler and code

Summary

Remove old personally_procured_move handler and endpoint.  Verified code is no longer being called from the application.

Verification Steps for Reviewers

These are to be checked by a reviewer.


How to test

1. Login as a customer, create a new PPM.
2. Login as a SC, fill in the required data, submit.
3. Back as a customer, upload documents, etc., submit back to the SC.
4. Test the entire process for the PPM to make sure all still works the same as expected.
